docs/root/intro/arch_overview/observability/access_logging.rst

```diff
@@ -17,6 +17,26 @@ logs<envoy_v3_api_field_config.listener.v3.Listener.access_log>`. The listener a
 HTTP request access logging and can be enabled separately and independently from
 filter access logs.
 
+.. _arch_overview_access_log_periodic:
+
+Periodic access logs
+--------------------
+
+If access log is enabled, then by default it will be flushed to the configured sinks at the end of a TCP
+stream, or HTTP request. It is possible to extend this behavior and flush access logs periodically, and
+at the beginning of a TCP stream.
+
+TCP Proxy
+*********
+
+* Periodic access logs can be enabled using :ref:`access log flush interval <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.access_log_flush_interval>`
+* Access log after upstream connection can be enabled using :ref:`flush access log on connected <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.flush_access_log_on_connected>`
```

Comment:
Can you rephrase to make it more clear that it is right after, not just after? Maybe ```access log on the upstream connection establishment```, and also clarify if this requires periodic reporting or not.

Reply:
done